packages/tradershub/src/features/cfd/screens/DynamicLeverage/DynamicLeverageTitle.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+import { Text } from '@deriv/quill-design';
+import { StandaloneArrowLeftBoldIcon } from '@deriv/quill-icons';
+import { useDynamicLeverageModalState } from '../../components/DynamicLeverageContext';
+
+export const DynamicLeverageTitle: FC = () => {
+    const { toggleDynamicLeverage } = useDynamicLeverageModalState();
+
+    return (
+        <div className='flex items-center px-0 border-b gap-800 pl-800 pr-1200 h-14 border-system-light-secondary-background '>
```

Comment:
`px-0` doesn't work here. `@deriv/quill-design` overwrites all of the tailwind base spacings and colors. The correct one is `px-50`.

Reply:
You can refer to the `tailwind.config.js` from the `@deriv/quill-design`

Reply:
Updated.